upgrade/upgrade-scripts/rootfs-cleanup

```diff
@@ -110,7 +114,18 @@ def main() -> None:
     # will always increase (or remain constant, in the case of rollback)
     # after an upgrade (i.e. we never upgrade to a lower version).
     #
-    for rootfs in sorted(filesystems + snapshots, key=rootfscmp)[:-2]:
+    # Snapshots must be deleted before filesystems because the following code
```

Comment:
so, I don't think the order that we concatenate the two lists together matters, since we iterate over the combination in version sorted order..

so even if we change this to `snapshots + filesystems`, we'll probably delete a filesystem before any snapshots on it, due to the filesystem generally having a greater version than the snapshots on the filesystem..

I could be wrong, as I haven't verified, though.. regardless, I think with the `exists` check, the order shouldn't matter..

Reply:
fixed